src/rust/catloop/futures/connect.rs

```diff
@@ -46,7 +46,7 @@ enum ClientState {
     InitiateConnectRequest,
     ConnectRequestSent(QToken),
     ConnectAckReceived(QToken),
-    Connected(SocketAddrV4, Rc<DuplexPipe>),
+    Connected(SocketAddrV4, Rc<DuplexPipe>, QToken),
```

Comment:
Perhaps move this to the beginning to match the order of other enums like ServerState.

Reply:
Sure!